jekyll/_data/api.yml

```diff
@@ -389,6 +389,20 @@ add_user_ssh:
     This allows them to SSH to the build containers.
   response: '{... the build data ... }'
 
+project_build:
+  url: "/api/v1.1/project/:vcs-type/:username/:project/build"
+  description: "Trigger a build of the specified project, by branch / revision or tag. All workflows found in the build configuration will be run."
```

Comment:
"All workflows" may be misleading in that scheduled workflows won't run - may make more sense to say something like: "Workflows will be run or scheduled in the same way as when a webhook from source control is received"

Reply:
replaced